uint8         startChar
uint8[]       cmd
uint8[]       mTime
float64       lat
uint8         NorS
float64       lon
uint8         EorW
uint8[]       mode
int32         count
float64       horizonprecious 
float64       geoidalheight
uint8         meter1
float64       height
uint8         meter2
float64       dataage
int32         basestationflag
uint8         sperator
uint8[]       checksum